docs/user/security/securing-kibana.asciidoc

```diff
@@ -74,20 +73,21 @@ For more information, see <<xpack-security-session-management, Session managemen
 
 . Restart {kib}.
 
-. [[kibana-roles]]Choose an authentication mechanism and grant users the privileges they need to
-use {kib}.
+. Temporarily log into {kib} using the built-in `elastic` superuser so you can create new users and assign roles. If you are running {kib}
+locally, go to `https://localhost:5601` to view the login page.
 +
---
-For more information on Basic Authentication and additional methods of
-authenticating {kib} users, see <<kibana-authentication>>.
+NOTE: The password for the built-in `elastic` user is typically set as part of the security configuration process on {es}. For more
+information, see {ref}/built-in-users.html[Built-in users].
 
-To manage privileges, open the main menu, then click *Stack Management > Roles*.
+. [[kibana-roles]]Create roles and users to grant access to {kib}.
++
+--
+To manage privileges in {kib}, open the main menu, then click *Stack Management > Roles*. The built-in `kibana_admin` role will grant
```

Comment:
I made two changes here:

* Added a new bullet that clarifies the user should temporarily log into Kibana as the superuser
* Changed the "Choose an authentication mechanism and grant users the privileges they need to use {kib}." bullet to "Create roles and users to grant access to {kib}.", keeping the anchor intact

It seems to me that "Choose an authentication mechanism" is out of place here and not really appropriate for a quick-start guide.

Reply:
I like both of these changes, and I agree that choosing an authentication mechanism is a bit out of place